ngi_pipeline/engines/piper_ngi/utils.py

```diff
@@ -41,6 +41,35 @@ def launch_piper_job(command_line, project, log_file_path=None):
         log_process_non_blocking(popen_object.stderr, LOG.warn)
     return popen_object
 
+def spawn_child_md5(project):
+    """Spawns deatached child processes that calculates 
+    md5sum on the bam files in dir 05 and .genomic.vcf files
+    in dir 07.
+    """
+    path=locate_project(project, 'ANALYSIS')
+    fileList05=glob.glob("{}/piper_ngi/05_processed_alignments/*".format(path))
+    fileList07=glob.glob("{}/piper_ngi/07_variant_calls/*".format(path))
+
+    for item in fileList05:
+        if item.endswith((".bam",".bai",".metrics")):
+            if os.path.isfile("{}.md5".format(item)):
+                continue
+            elif os.path.isfile("{}.part".format(item)):
+                continue
+            else:
+                command=["nohup bash -c \" md5sum {0} | cut -f 1 -d ' '  > {0}.part; mv {0}.part {0}.md5\" > /dev/null 2>&1".format(item)] 
```

Comment:
Try to give a look here https://github.com/SciLifeLab/TACA/blob/master/taca/utils/misc.py#L31
this function executes an external program/command and takes extra care of std err and std out instead of sending to /dev/null . In this way we can have better logging of md5 computation